src/appleseed/renderer/modeling/aov/pixeltimeaov.cpp

```diff
@@ -181,38 +183,11 @@ namespace
         void post_process_image(const Frame& frame) override
         {
             const AABB2u& crop_window = frame.get_crop_window();
+            ColorMap color_map;
```

Comment:
Did you test this code? As far as I can tell, it won't work anymore since you're not loading a palette.

Please make sure you test all the code you modified.

Reply:
Yes, I tested this code for all three AOVs and Color map post-processing stage.
Moreover, this part of the code hasn't been changed _logically_ from the previous round of review. We didn't set the palette then and we didn't set it now. In fact, I even included a comment in `evaluate_palette` method regarding this.
`// When image contains a single channel we do not set the color palette.
    if (m_palette.empty())
        return Color3f(x);`

If you were expecting a segmentation fault because of this line `m_palette.empty()`, I tested it in a dummy C++ program, it worked so I put it there. I think this is because of `std::vector`'s default constructor gets called.

Moreover, the way things are implemented now in `colormappostprocessingstage.cpp`, we have a private member variable (`m_palette`) which is a `vector<Color3f>` and isn't explicitly initialized anywhere but we do call `resize()` on it when we set the palette (`set_palette_from_array`, `set_palette_from_image_file`) which I think is not much different than this line `m_palette.empty()`.

Sorry, but is there something I'm not getting?

I'm too anxious to push untested code even if the only thing a I change is a comment. 😅